### DISCUSSION TITLE
Mode to Disable Bots

### DISCUSSION THREAD

**Discussion by lambda-11235** [redeclipse/base#1137] [Ideas and Feature Requests]
It's easy to notice that survivor mode is a community favourite. However, it has come to my that this may be in part because survivor mode disables bots. This is unfortunate because survivor mode does entail that less skilled players (such as myself some times) spend a majority or their time spectating, especially when playing with more skilled players.

My suggestion would be to add a mode that disables bots. This addresses the issue of avoiding bots without having players go through a lot of downtime.

An alternative is a mode to allow only 1 bot (or 3 in multi) in the game, so that teams are balanced without being overrun by bots.

**Comment by lambda-11235** [comment]
Also, /botbalance doesn't work for non-admins, that's why I'm suggesting a new mode.

**Comment by qreeves** [comment]
What I'd ultimately like to do is have a third kind of game modifier (after mode and mutator) that accepts a list of `[variable value]` pairs, then include a list of the most common custom settings (like influencing bot balance).

I want to give players more control over the game they play so they can keep it interesting for themselves, but allowing variable changes as a "free-for-all" isn't a very good solution, which is why I think limiting the changes to the map votes themselves should keep things coherent.

At the very least, worth an experiment, otherwise we can just chuck in a mutator or two to control the bot balance behaviour. Any other thoughts from the community?
